keyboards/ky01/keymaps/default/keymap.c

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2020 KnoblesseOblige
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define ___ KC_TRNS
+
+enum layer_names {
+    QWERTY,
+    L1
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [QWERTY] = LAYOUT(
+        KC_ESC,            KC_F1,   KC_F2, KC_F3, KC_F4,           KC_F5, KC_F6, KC_F7,    KC_F8,   KC_F9,    KC_F10,   KC_F11,   KC_F12,    LT(L1, KC_PSCR),
+        KC_GRAVE, KC_1,    KC_2,    KC_3,  KC_4,  KC_5,  KC_6,     KC_7,  KC_8,  KC_9,     KC_0,    KC_MINUS, KC_EQUAL, KC_BSPC,  KC_DELETE, KC_INS,
+        KC_TAB,   KC_Q,    KC_W,    KC_E,  KC_R,  KC_T,  KC_Y,     KC_U,  KC_I,  KC_O,     KC_P,    KC_LBRC,  KC_RBRC,  KC_BSLS,             KC_DELETE,
+        KC_CAPS,  KC_A,    KC_S,    KC_D,  KC_F,  KC_G,  KC_H,     KC_J,  KC_K,  KC_L,     KC_SCLN, KC_QUOT,  KC_NUHS,  KC_ENTER,            KC_PGDN,
+        KC_LSFT,  KC_NUBS, KC_Z,    KC_X,  KC_C,  KC_V,  KC_B,     KC_N,  KC_M,  KC_COMMA, KC_DOT,  KC_SLSH,  KC_RSFT,  KC_UP,
+        KC_LCTL,  KC_LGUI, KC_LALT,                      KC_SPACE,               KC_RGUI,  KC_RALT, KC_RCTL,  KC_LEFT,  KC_DOWN,  KC_RIGHT
+    ),
+    [L1] = LAYOUT(
+        RESET,    ___, ___, ___, ___,      ___, ___, ___, ___, ___, ___, ___, ___, ___,
+        ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___,
+        ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___,      ___,
+        ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___,      ___,
+        ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___, ___,
+        ___, ___, ___,                ___,           ___, ___, ___, ___, ___, ___
```

Comment:
```suggestion
        RESET,   _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,                            _______,                   _______, _______, _______, _______, _______, _______
```

Reply:
Suggestion applied in commit https://github.com/qmk/qmk_firmware/pull/12844/commits/45da15b3923e998097f6368e754b11232dc655dd